### DISCUSSION TITLE
SEF permalink Profile URLs

### DISCUSSION THREAD

**Discussion by joomleb** [joomla/joomla-cms#32086] [2. Cold Features]
Hi guys,
I'm posting a discussion here thanks to Alikon suggestion on this "[Feature Request](https://github.com/joomla/joomla-cms/issues/32080)"
Considering we now have Custom Fields to complete the Joomla User Profile, to let users to have SEF permalink Profile URLs to their Profile page like: https://site.com/John-Ping is really a must on Joomla 4

Till few months ago Mike Feng has developed for years a sinple plugin for it: https://www.simbunch.com/products/profileurl
But, I don't know what happened to him, it's been 9 months since he literally disappeared !
(Please, Do anyone know anything about Mike?)

**Comment by brianteeman** [comment]
ITs a lot lot more than simply using a custom field

**Comment by bembelimen** [comment]
See: https://github.com/joomla/joomla-cms/pull/21441

**Comment by joomleb** [comment]
Hi guys, really many thanks for your answers. I studied a lot into them and I want to revamp this discussion from another point of you to reach at the same goal in a more simple, less maintenance effort, more open to all solutions...

The "[SEBLOD User Bridge](https://www.seblod.com/resources/tutorials/using-the-seblod-user-bridge)" is a good suggestion, [here a waited enhancement](https://github.com/Octopoos/SEBLOD/issues/309)

I think it is time for Joomla to treat Users just as an Article into a default "Users" Category, where on top of the Article structure are added ("as Custom Fields") the necessaries default Users Tabs "Account Details", "Assigned Users Groups" etc. ...
Some of the advantages:
- more simple standard layout
- all the 3rd party features extensions for articles are yet for Users (Searches, Tags, Open Graphs, Mapping, ...etc...)
- this will be true also for all the Searches and Filters features
- Users will take the advantage of all the Articles features like publish/unpublish
- For developers will be more simple, less maintenance developers effort, 
- more open to all solutions...
- The SEF URL would be solved, will be the standard articles one
- The "Contact" component could be cut out from the standard Joomla release without loose any feature (less developers work) 
- ...
